docs/api/en/helpers/BoxHelper.html

```diff
@@ -28,10 +28,32 @@ <h2>Example</h2>
 
 
 		<code>
-		var sphere = new THREE.SphereGeometry();
-		var object = new THREE.Mesh( sphere, new THREE.MeshBasicMaterial( 0xff0000 ) );
-		var box = new THREE.BoxHelper( object, 0xffff00 );
-		scene.add( box );
+		// Creating the object whose bounding box we want to compute
```

Comment:
Sorry but I think the previous code example was totally sufficient. `BoxHelper` is just a helper class and the example code should demonstrate its creation. The actual handling of a `Box3` does not belong to this page.

Reply:
Yeah it makes sense, I guess I should rather add a decent example to Box3 as it doesn't have one and its workflow isn't that immediate.

Reply:
Correct.